### DISCUSSION TITLE
Help! Stuck on STEP 4 last step with the output. Can someone please advise what the expected output is supposed to look like?

### DISCUSSION THREAD

**Discussion by anieshak** [skills/.github#607] [Q&A] [answered]
I am getting error - sl: command not found while trying to execute step 4 of the Activity - 
**Code with GitHub Codespaces and Visual Studio Code**

![sl command not found](https://github.com/skills/.github/assets/20052332/ae95647b-681a-46b2-a57b-9d47122fcde1)
 

Step 14 - **Verify your codespace is running, as you did previously.** - I did

Step 15 - **Verify the setup.sh file is present in your VS Code editor.**  - I did

**< I was able to verify that the above files were present yet the command (sl) doesn't seem to function.>**

Step 16 - **From the VS Code terminal, type or paste:**

**sl**
**Enjoy the show!**

_I am unable to realize what this command is supposed to do? Can someone please help as to what the output should be. I wish Github pasted screenshots of what the expected output was supposed to look like._

**Comment by RachBreeze** [comment]
Hello @anieshak

I too am experiencing this issue and  I believe this is related to the issue raise here https://github.com/skills/code-with-codespaces/issues/121.

**Comment by RachBreeze** [comment, marked as answer]
Hello @anieshak 

Further investigation showed the issue was with the bash script. I updated setup.sh as follows, note the` -y`:

#!/bin/bash

```
sudo apt-get update
sudo apt-get install -y sl
echo "export PATH=\$PATH:/usr/games" >> ~/.bashrc
```

**Reply by anieshak** [reply]
Appreciate you taking time to respond 😊
